src/Azs.ContainerRegistry.Admin/docs/Start-AzsContainerRegistrySetup.md

```diff
@@ -72,6 +74,21 @@ Returns the error if another instance of Setup is already started.
 
 ## PARAMETERS
 
+### -CertificateInputFile
+Input File for Certificate (Ssl certificate in base64 format.)
+
+```yaml
+Type: System.String
+Parameter Sets: StartExpanded, StartViaIdentityExpanded
+Aliases:
+
+Required: False
```

Comment:
Whys is this false ?. Isn't the password and certfile should be mandatory ?.

Reply:
Need to update swagger spec to make the params mandatory, doing this.

Reply:
Can u then please open a tracking bug for this. So that we can update it later.